build/release.js

```diff
@@ -27,6 +29,15 @@ module.exports = function( Release ) {
 			ensureSizzle( Release, callback );
 		},
 		/**
+		 * Set the version in the src folder for distributing AMD
+		 */
+		_setSrcVersion: function() {
+			var corePath = __dirname + "/../src/core.js",
+				contents = fs.readFileSync( corePath, "utf8" );
+			contents = contents.replace( /@VERSION/g, Release.newVersion );
```

Comment:
Somewhere i suggested to use underscore templates since they used in grunt itself and lodash (obviously) also has them, which make them kind of standard in some sense.

The bigger question for me is that we have a lot of custom logic for the build, i don't know, but it seems this should be a common issue and therefore should have a common solution...

But of course i need to put my money where the mouth is, i.e. do something about this, otherwise it is just unproductive critic from my side.